src/common/components/elements/FeaturedListingsCarousel.tsx

```diff
@@ -0,0 +1,57 @@
+import { Row, Carousel } from 'antd';
+import { Listing, ListingPreview } from './ListingPreview';
+
+// still some style cleanup to do here.... :(
+export function FeaturedListingCarousel(props: { featuredListings: Listing[] }) {
+  return (
+    <div style={{ maxWidth: 500 }}>
+      <div
+        style={{
+          position: 'relative',
+          display: 'flex',
+          justifyContent: 'space-between',
+          marginBottom: '1rem',
+        }}
+      >
+        <span
+          style={{
+            color: 'white',
+            opacity: 0.6,
+            fontSize: 14,
+            fontWeight: 400,
+          }}
+        >
+          Featured listings
+        </span>
```

Comment:
Look to use text component with props like strong to build the text.

Reply:
Solved